google/cloud/storage/transfer_manager.py

```diff
@@ -244,21 +245,17 @@ def download_many(
     :type blob_file_pairs: List(Tuple('google.cloud.storage.blob.Blob', IOBase or str))
     :param blob_file_pairs:
         A list of tuples of blob and a file or filename. Each blob will be
-        downloaded to the corresponding blob by using blob.download_to_file() or
-        blob.download_to_filename() as appropriate.
+        downloaded to the corresponding blob by using blob._prep_and_do_download() or blob._handle_filename_and_download() as appropriate.
```

Comment:
Hmm, this is the tricky part. We want to avoid clients having to peek at private methods if possible. It's a little awkward, but how about this:

"downloaded to the corresponding blob by using APIs identical to blob.download_to_file() or blob.download_to_filename() as appropriate"

Additionally keep using blob.download_to_filename() in the "note" below.

Then, don't change the download_kwargs section at all, because the arguments should be identical.

The intention here is to keep the users focused on public methods, and also to keep the private methods functionally identical to the public methods wherever possible.